.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  root: true,
+  env: {
+    browser: true,
+    node: true
+  },
+  extends: [
+    '@nuxtjs/eslint-config-typescript',
+    'plugin:nuxt/recommended',
+    'prettier'
+  ],
+  plugins: [
+  ],
+  // add your custom rules here
+  rules: {}
```

Comment:
Can we add sort-keys rules while we are on it? since we talked about it for tsconfig stuff https://eslint.org/docs/rules/sort-keys